IpmiFeaturePkg/GenericIpmi/Common/GenericIpmi.h

```diff
@@ -6,8 +6,8 @@
   SPDX-License-Identifier: BSD-2-Clause-Patent
 **/
 
-#ifndef _IPMI_BMC_H_
-#define _IPMI_BMC_H_
+#ifndef _GENERIC_IPMI_H_
+#define _GENERIC_IPMI_H_

```

Comment:
nit: Kun had recommended to me a format of `Generic_IPMI_H_`

Reply:
Ah, gotcha. Looks like a ton of EDKII uses the leading '_', but looking at the coding standards it does not. Will fix.